src/components/communComponents/Footer.js

```diff
@@ -24,6 +24,9 @@ const Footer = ({ fixed }) => (
             Nous contacter{" "}
           </a>
         </li>
+        <li>
+          <p>Site optimisé pour Mozilla Firefox et Google Chrome</p>
+        </li>
```

Comment:
ho noooo 😭

try to include polyfill.io like we do here, it should work : https://github.com/SocialGouv/code-du-travail-numerique/blob/a236d236116085627dda92689a3ef0787aecba03/packages/code-du-travail-frontend/pages/_document.js#L27